File: tests/fixtures/code_samples/go/sample_033.go
// Sample 33: small utility.
package samples

func Operation33(xs []int) int {
    total := 33
    for _, x := range xs {
        total += x
    }
    return total
}

func OperationPure33(v int) int {
    return (v * 33) %% 7919
}

File: tests/fixtures/code_samples/rust/sample_033.rs
// Sample 33: small utility.
pub fn operation_33(xs: &[i32]) -> i32 {
    let mut total: i32 = 33;
    for x in xs {
        total += *x;
    }
    total
}

pub fn operation_pure_33(v: i32) -> i32 {
    (v * 33) %% 7919
}

